admin-frontend/open_admin_app/lib/widgets/features/features_overview_table_widget.dart

```diff
@@ -22,42 +24,38 @@ class FeaturesOverviewTableWidget extends StatelessWidget {
   Widget build(BuildContext context) {
     final bloc = BlocProvider.of<PerApplicationFeaturesBloc>(context);
 
-    try {
-      return StreamBuilder<FeatureStatusFeatures?>(
-          stream: bloc.appFeatureValues,
-          builder: (context, snapshot) {
-            if (!snapshot.hasData) {
-              return const SizedBox.shrink();
-            }
-
-            if (snapshot.hasData &&
-                snapshot.data!.sortedByNameEnvironmentIds.isEmpty) {
-              return Column(
-                crossAxisAlignment: CrossAxisAlignment.start,
-                children: const <Widget>[
-                  NoEnvironmentMessage(),
-                ],
-              );
-            }
-            if (snapshot.hasData &&
-                snapshot.data!.applicationFeatureValues.features.isEmpty) {
-              return const NoFeaturesMessage();
-            }
-
-            if (snapshot.hasData) {
-              return TabsView(
-                featureStatus: snapshot.data!,
-                applicationId: bloc.applicationId!,
-                bloc: bloc,
-              );
-            } else {
-              return const NoFeaturesMessage();
+    return StreamBuilder<FeatureStatusFeatures?>(
```

Comment:
if you could please remove this change, its not wrong, but it I am doing some extensive refactoring in this area and this has all changed, and i don't want to have to unmangle it :-)